stakedb/ticketpool.go

```diff
@@ -324,183 +252,75 @@ func NewTicketPool(dataDir, dbSubDir string) (tp *TicketPool, err error) {
 	}, nil
 }
 
-// LoadAllPoolDiffs loads all found ticket pool diffs from badger DB.
-func LoadAllPoolDiffs(db *badger.DB, ver uint32) ([]PoolDiff, []uint64, error) {
```

Comment:
This method is exported but I don't see a need to have it exported if `NewTicketPool` will do all the hard work(upgrade and load diffs).